xinference/model/llm/tests/test_utils.py

```diff
@@ -240,6 +240,37 @@ def test_prompt_style_qwen():
     )
 
 
+def test_chatML_style():
+    prompt_style = PromptStyleV1(
+        style_name="CHATML",
+        system_prompt="<system>{system_message}\n",
```

Comment:
System prompt should not be an f-string.

Reply:
change to a customized message in the latest version